cleansweep/views/admin.py

```diff
@@ -283,3 +283,18 @@ def _load_contacts(place, data):
         contacts += p.add_contacts(prows)
     db.session.commit()
     return contacts
+
+
+@app.route("/admin/settings/export", methods=['GET'])
```

Comment:
Are we gonna have separate endpoint for each export?
I was thinking to do it just once in admin settings. Like mentioned in #182.